src/elements/select/select.visual.spec.ts

```diff
@@ -194,6 +196,20 @@ describe('sbb-select', () => {
           }),
         );
 
+        it(
+          `negative=${negative} multiple=${multiple} size=s`,
+          visualDiffDefault.with(async (setup) => {
+            await setup.withFixture(template({ ...defaultArgs, negative, multiple, size: 's' }), {
+              minHeight: '600px',
+              backgroundColor: negative ? 'var(--sbb-color-black)' : undefined,
+            });
+            setup.withPostSetupAction(() => {
+              const select = setup.snapshotElement.querySelector('sbb-select')!;
```

Comment:
I think there should be a selected option to show placement of the label in the form field size s

Reply:
Just pushed a fix for it